v2/README.md

```diff
@@ -27,6 +27,15 @@ Sample YAMLs for creating each of these resources can be found in the [samples d
     ```bash
     kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.1.0/cert-manager.yaml
     ```
+   Check that the cert-manager pods have started successfully before continuing.
```

Comment:
As mentioned in my other comment you can use `cmctl` here if you want as well: `cmctl check api --wait=2m`

Reply:
Do we want to depend on `cmctl` in our installation instructions? I wouldn't have thought we'd want people to need to install it just for this?

Reply:
I wouldn't want to depend on it, no - but if we want we could mention it as an alternative way to check? I was just thinking something like: "Optionally instead use `cmctl check api --wait=2m`, see X for more details about cmctl"